common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for the bulk_user_activate command."""
+
+import ddt
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from student.tests.factories import UserFactory
+from student.models import User
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+
+from student.models import get_users_by_email
+
+
+@ddt.ddt
+class BulkUserActivateTests(SharedModuleStoreTestCase):
+    """Tests for the bulk_user_activate command."""
+
+    def setUp(self):
+        super(BulkUserActivateTests, self).setUp()
+        self.users = UserFactory.create_batch(5)
+
+    def _count_num_active(self, user_list):
+        num_active = 0
+        for user in user_list:
+            if user.is_active:
+                num_active += 1
+        return num_active
+
+    def _get_domain_size(self, user_list, domain):
+        size = 0
+        dom_users = get_users_by_email(domain)
+        for user in dom_users:
+            size += 1
+        return size
+
+    @ddt.data('stanford.edu', 'example.com', 'fake.email')
+    def test_bulk_without_force(self, domain):
+        """Verify that nothing is changed when force is set to false."""
+        prev_num_active = self._count_num_active(self.users)
+
+        call_command(
+            'bulk_user_activate',
+            domain=domain,
+            force=False,
+        )
+
+        self.assertEqual(self._count_num_active(self.users), prev_num_active)
+
+    @ddt.data('stanford.edu', 'example.com', 'fake.email')
+    def test_bulk_with_force(self, domain):
```

Comment:
I think there are some problems with this test that will come to light when you add the filter I mentioned in a previous comment. Also, what is the is_active state of a user after creation by UserFactory? If it is 1 then you haven't tested anything here.